esm-collection/json-schema/schema.json

```diff
@@ -1,14 +1,15 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "schema.json#",
   "title": "ESM Extension Specification",
   "description": "This object represents Collections in a Earth System Model Catalog.",
   "allOf": [
     {
-      "$ref": "https://raw.githubusercontent.com/radiantearth/stac-spec/v1.0.0-beta.1/collection-spec/json-schema/collection.json"
+      "$ref": "#/definitions/catalog"
```

Comment:
Catalog is already included in Collection, so the ref to catalog can be removed.